apple-app-site-association.json

```diff
@@ -31,13 +33,15 @@
           "NOT /click/*/aHR0cHM6Ly9hcnRzeS50eXBlZm9ybS5j*",
           "NOT /click/*/aHR0cHM6Ly9hcHAuYWRqdXN0LmNv*",
           "NOT /users/auth/facebook/callback*",
+          "NOT /identity_verification*",
           "NOT *L2FydGljbGUv*",
           "NOT *L3ZpZGVv*",
           "NOT *L25l*",
           "NOT *L25ld3Mv*",
           "NOT *LzIwMTYteWVhci1pbi1h*",
           "NOT *L3ZlbmljZS1iaWVubmFs*",
           "NOT *L2dlbmRlci1lcXVhbGl0*",
+          "NOT *L2lkZW50aXR5X3ZlcmlmaWNhdGlv*",
```

Comment:
Why is it necessary? I'm a bit confused as to whether we only want to have an entry here for lines that start with [`NOT /click`](https://github.com/artsy/artsy-eigen-web-association/pull/32/files#diff-6f6158418e77656d6b546b8bef9810bfL33) OR for all lines... cc @ashfurrow

Reply:
This is to cover Sailthru links. Sailthru base64 encodes URLs, and this line reflects the added `/identity_verification*` route. Otherwise, marketing emails that link to `link.artsy.net/LzIwMTYteWVhci1pbi1hiasuv932nv8s8` for a collection or whatever don't route into the app. I know it's unlikely that marketing will send out identity verification links, but [the script](https://github.com/artsy/artsy-eigen-web-association/blob/master/add-sailthru-paths.js) adds them anyway. Let me know what I can clarify.

Reply:
Got it – I then wonder if [L33-34](https://github.com/artsy/artsy-eigen-web-association/pull/32/files/ebdfa775597ba4f122e3512c5dce505316bc8b8b#diff-6f6158418e77656d6b546b8bef9810bfL31-R34) were created the mentioned script was created/ran, and maybe could be removed.

Probably not worth the risk at this point, but wanted to call out anyways.